packages/inter-protocol/src/auction/util.js

```diff
@@ -17,10 +14,17 @@ export const AuctionState = {
   WAITING: 'waiting',
 };
 
-export const makeBrandedRatioPattern = (nBrand, dBrand) => {
+/**
+ * @param {Pattern} numeratorAmountShape
+ * @param {Pattern} denominatorAmountShape
+ */
+export const makeBrandedRatioPattern = (
```

Comment:
this function isn't doing much at this point. if you want it to still take brands it could use https://github.com/Agoric/agoric-sdk/pull/7094/commits/530ed9c932372ca237cc4502a17d2393e3a31cb9

if you keep the current signature, could be worth statically verifying the pattern:
```js
 * @param {{ brand: Brand, value: Pattern }} numeratorAmountShape
 * @param {{ brand: Brand, value: Pattern }} denominatorAmountShape
```

Reply:
I changed the parameter declarations, but left the rest alone.